### ISSUE TITLE
No way to upload proof of academic status for education benefits

### ISSUE THREAD

**Issue by joel-coffman** [github/docs#19342]
### Code of Conduct

- [X] I have read and agree to the GitHub Docs project's [Code of Conduct](https://github.com/github/docs/blob/main/CODE_OF_CONDUCT.md)

### What article on docs.github.com is affected?

https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/use-github-in-your-classroom-and-research/apply-for-an-educator-or-researcher-discount

### What part(s) of the article would you like to see updated?

The [instructions for applying for an educator or researcher discount](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/use-github-in-your-classroom-and-research/apply-for-an-educator-or-researcher-discount#applying-for-an-educator-or-researcher-discount) include a way to upload proof of academic status, but the (current) form does not have an upload button -- just the ability to take a picture using the computer's camera. See the screenshot in "Additional information" (below), which shows the missing upload button.

### Additional information

<img width="1436" alt="Screen Shot 2022-07-27 at 10 39 46 AM" src="https://user-images.githubusercontent.com/8376231/181304972-30a9e57d-2421-45d0-a215-eac1432017d1.png">

**Comment by cmwilson21** [comment]
@joel-coffman Thanks for opening an issue and linking it to your PR! ⚡

**Comment by joel-coffman** [comment]
From a response to a GitHub support ticket:

> Sorry for the inconvenience. It seems that you have applied from a location that is far from your campus. You won't get an option to upload because of how far you are from the campus.

I was able to confirm that restriction, as I tried again using a different commercial Internet connection, and lo-and-behold, the upload button did appear!

Perhaps instead of removing the note, the restriction / limitation should documented? I'm also curious how the geolocation is implemented, as (ironically) the upload button did **not** appear when connected to the institution's WiFi network but did when connected to an independent commercial network.
